js/src/routes.js

```diff
@@ -16,11 +16,15 @@
 
 import {
   Accounts, Account, Addresses, Address, Application,
-  Contract, Contracts, Dapp, Dapps,
+  Contract, Contracts, Dapp, Dapps, HistoryStore,
```

Comment:
My knee-jerk reaction was that `~/util` is even worse, my thoughts after 30s is that it _may_ fit. (Although it really is meant for swiss-army-knife utilities)